weblate_web/templates/snippets/reviews.html

```diff
@@ -0,0 +1,77 @@
+{% load i18n %}
+{% load static %}
+
+<div>
+   <a href=""><img src="{% static "img/Siemens-logo.png" %}" alt="Siemens" /></a>
+   <a href=""><img src="{% static "img/Dyson-logo.png" %}" alt="Dyson" /></a>
+   <a href=""><img src="{% static "img/YouGov-logo.png" %}" alt="YouGov" /></a>
+</div>
+<br>
+<div class="simple-slider page-slider" dir="ltr">
+    <div class="slider-wrapper" dir="{% if LANGUAGE_BIDI %}rtl{% else %}ltr{% endif %}">
+      <div class="slider-slide is-bright">
+        <div class="testimonial">
+            <div class="icon-review">
+                <img src="{% static "img/Quotes.png" %}" alt="Quotes" />
+                <p>Weblate is gorgeous app! It really saved my life. Translations are managed within same version control as code. Weblate is gorgeous app! It really my life. Translations are managed within.</p>
+            </div>
+            <div class="icon-user">
+                <div class="circle-user">
+                    <img src="{% static "img/donald.png" %}" alt="Quotes" />
+                </div>
+                <div class="contentt">
+                    <p>Donald McCalister</p>
+                    <a href=""><p>Dyson Electronics</p></a>
+                </div>
+            </div>
+            
+        </div>
+        </div>
+
+
+<div class="slider-slide">
```

Comment:
Can you please use proper indentation in the code? This makes it really hard to review when matching divs are not aligned.

Reply:
I have now, please check.